Gemfile.lock

```diff
@@ -162,15 +162,15 @@ GEM
     equatable (0.6.1)
     erubi (1.9.0)
     erubis (2.7.0)
-    faraday (0.17.0)
+    faraday (0.17.1)
       multipart-post (>= 1.2, < 3)
     faraday_middleware (0.12.2)
       faraday (>= 0.7.4, < 1.0)
     fauxhai-ng (7.5.1)
       net-ssh
-    ffi (1.11.2)
-    ffi (1.11.2-x64-mingw32)
-    ffi (1.11.2-x86-mingw32)
+    ffi (1.11.3)
```

Comment:
this seems to be failing?

Reply:
I tried a number of things to allow the FFI upgrade to work in those centos containers, but I'm kinda stuckk at this point. Once we get this merged the next build should be good.